gateways/adyen/adyen.go

```diff
@@ -65,10 +65,18 @@ func (client *AdyenClient) Authorize(request *sleet.AuthorizationRequest) (*slee
 	if result.AdditionalData != nil {
 		values, ok := result.AdditionalData.(map[string]interface{})
 		if ok {
-			response.AvsResult = translateAvs(values["avsResult"].(string))
-			response.CvvResult = translateCvv(values["cvcResult"].(string))
-			response.AvsResultRaw = values["avsResultRaw"].(string)
-			response.CvvResultRaw = values["cvcResultRaw"].(string)
+			if avs, isPresent := values["avsResult"]; isPresent {
```

Comment:
Avs and Cvc aren't returned for Apple Pay transactions so had to change this to prevent panics